components/dashboard/src/teams/sso/OIDCClients.tsx

```diff
@@ -66,7 +68,7 @@ const OIDCClientsList: FC<OIDCClientsListProps> = ({ clientConfigs }) => {
                         <ItemField>Issuer URL</ItemField>
                     </Item>
                     {clientConfigs.map((cc) => (
-                        <OIDCClientListItem key={cc.id} clientConfig={cc} />
+                        <OIDCClientListItem key={cc.id} clientConfig={cc} hasActiveConfig={hasActiveConfig} />
```

Comment:
**question**: This is a question to ask later, but do we want to allow multiple SSO configurations with the same Issuer URL?

<img width="1040" alt="Screenshot 2023-05-16 at 23 12 02" src="https://github.com/gitpod-io/gitpod/assets/120486/ccd073ce-cba6-4c20-a8b1-c5439225bc5f">

Reply:
Yes.

The issuer URI isn't longer considered a stable ID. 

Why is this useful? It's a combination of two things which spans a safety net for dealing with the SSO config of an Org:
 * We want to make sure an activated config isn't modified.
 * If you _need_ to update the config (a common use case be "client secret changed") we nudge you to create a sibling configuration for the same IdP, having the same coordinates, but updated attributes. 
   * After verification of that config, you'd be able to switch to it by activating, so that the Org accounts get/remain accessible.

Reply:
Thanks for the insights, @AlexTugarev! 🙇